cheddar/src/lib.rs

```diff
@@ -81,9 +82,10 @@ impl Contract {
         return self.owner_id.clone();
     }
 
-    /// minters can mint more
+    /// Mints new tokens to the `account_id`.
+    /// Panics if the function is calle by a not registered minter.
     #[payable]
-    pub fn mint(&mut self, account_id: &AccountId, amount: U128String) {
+    pub fn ft_mint(&mut self, account_id: &AccountId, amount: U128String) {
```

Comment:
I use `ft_mint` instead of `ft_mint_to` because we have `ft_transfer` not `ft_transfer_to`.